Cargo.toml

```diff
@@ -14,27 +14,23 @@ license = "Apache-2.0"
 
 [features]
 optimize = ["cw-optimizoor"]
-chain-reg = ["chain-registry", "rand"]
```

Comment:
cosm-orc doesnt need a chain registry since its only for local integration testing.  

This feature was unused, so Im removing it from v3